go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/rerost/nrredigo
+
+go 1.13
+
+require (
+	github.com/gomodule/redigo v2.0.0+incompatible
+	github.com/newrelic/go-agent v2.2.0+incompatible
```

Comment:
please specify [`v2.1.0`](https://github.com/newrelic/go-agent/blob/master/CHANGELOG.md#210)

Reply:
I think you using v.2.1.0
https://github.com/izumin5210/nrredigo/blob/master/Gopkg.lock#L29

I believe `go mod init` 's converting

Reply:
👀 
And i think you should use v2.14.1.
Because you using `newrelic.DatastoreSegment`
https://github.com/newrelic/go-agent/commit/dd6aaea3dcaffcef06985ebe8ce1866c7e639752#diff-b532a1b7722a9d7c0c1401814b60ca45